### ISSUE TITLE
Trigger npm on composer install

### ISSUE THREAD

**Issue by vidurb** [miranj/craft#8]
I think this is a no-brainer; we can install both sets of packages in one fell swoop.

**Comment by rungta** [comment]
Good idea. Only caveat is we need this to happen only on `dev` environments.
